docs/tildagon-apps/development.md

```diff
@@ -434,7 +441,8 @@ You can test your app on-device, without publishing it, using [`mpremote`](https
     ```json
     {
         "name": "The OG Snake app",
-        "path": "apps.snake.app"
+        "path": "apps.snake.app",
+        "callable": "SnakeApp"
```

Comment:
This isn't the preferred way of doing this - as shown on line 36, apps should define `__app_export__` and point it to their app class.